### DISCUSSION TITLE
支持的交易产品品种问题

### DISCUSSION THREAD

**Discussion by lanshango820** [wondertrader/wondertrader#65] [Q&A] [answered]
请问是否支持加密货币交易呢？

**Comment by wondertrader** [comment, marked as answer]
wondertrader目前来说并不支持加密货币，暂时也没有计划加密货币的交易。如果作为研究的话，可以参考一下全天候行情的接入方式。wondertrader架构设计上相对来说比较灵活，你如果有这方面的需求，可以自行研究一下，看看通过一些修改是否可以支持
